components/ai_chat/resources/page/components/conversation_list/index.tsx

```diff
@@ -171,11 +133,14 @@ function ConversationList(props: ConversationListProps) {
                 <div
                   className={styles.message}
                 >
-                  {!turn.selectedText &&
-                      <FormattedTextRenderer text={turn.text} />}
+                  {!turn.selectedText ? (
```

Comment:
Just curious - why has this changed to a ternary `: null` instead of `&& ...` ? The behavior is the same